recipes/cuquantum/run_test.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -ex
+
+# integraty test
+test -f $PREFIX/include/custatevec.h
+test -f $PREFIX/include/cutensornet.h
+test -f $PREFIX/include/cutensornet/types.h
+test -f $PREFIX/lib/libcustatevec.so
+test -f $PREFIX/lib/libcutensornet.so
+
+# dlopen test
+${GCC} test_load_elf.c -std=c99 -Werror -ldl -o test_load_elf
+./test_load_elf $PREFIX/lib/libcustatevec.so
+./test_load_elf $PREFIX/lib/libcutensornet.so
+
+# compilation test
+git clone https://github.com/NVIDIA/cuQuantum.git sample_linux/
```

Comment:
Use `test: source_files` in `meta.yaml` or use a git tag here.

Reply:
```suggestion
git clone -b v0.1.0.0 https://github.com/NVIDIA/cuQuantum.git sample_linux/
```